packages/components-react/projects/nextjs/pages/table.tsx

```diff
@@ -1,15 +1,18 @@
 import { PTable, PTableHead, PTableHeadCell, PTableHeadRow } from '@porsche-design-system/components-react/ssr';
-import { TableWithCaption } from '../components';
+import { TableWithCaption, TableWithCaptionDark } from '../components';
 import type { NextPage } from 'next';
 
 const TablePage: NextPage = (): JSX.Element => {
   return (
     <>
-      <div className="playground light" title="should render table">
+      <div className="playground light" title="should render table with light mode">
```

Comment:
Let's stay consistent with other tests where the suffix is always `on light background` or `on dark background`.

Reply:
done